### ISSUE TITLE
Is It Possible to Use TAPE for Sequence Generation?

### ISSUE THREAD

**Issue by brucejwittmann** [songlab-cal/tape#81]
This is not an issue, but more a question about inbuilt functionality. Say I pass in a sequence with some residues masked. Is there a simple way to use the pre-trained TAPE models to return amino acid probabilities for each masked position?

**Comment by rmrao** [comment]
Sure - this should work:

```python
import torch
from tape import ProteinBertForMaskedLM
from tape.tokenizers import TAPETokenizer

model = ProteinBertForMaskedLM.from_pretrained('bert-base')
tokenizer = TAPETokenizer(vocab='iupac')

sequence = "<string input>"
tokens = list(sequence)
tokens[i] = tokenizer.mask_token  # Do this for as many positions as you want
ids = torch.tensor(tokenizer.convert_tokens_to_ids(tokens), dtype=torch.long).unsqueeze(0)

logits = model(ids)[0]
probs = logits.softmax(-1)
```
